lib/parsers/mp3_parser.rb

```diff
@@ -135,7 +135,8 @@ def parse_mpeg_frames(io)
     MAX_FRAMES_TO_SCAN.times do |frame_i|
       # Read through until we can latch onto the 11 sync bits. Read in 4-byte
       # increments to save on read() calls
-      data = io.read(4)
+      bites_to_read = 4
+      data = io.read(bites_to_read)
 
       # If we are at EOF - stop iterating
       break unless data && data.bytesize == 4
```

Comment:
Do you want to replace the `4` here as well, with the new variable?

Reply:
good! I took the oportunitty to do this in other places